### PR TITLE
[1.3.6-prepare][ambari]upgrade version

### PR DESCRIPTION
**Pull request by CalvinKirs**

### CODE REVIEW

**Comment by chengshiwen** [comment]
@CalvinKirs Due to the merged pr of docker/k8s, there are 3 conflicts 😂

File: ambari_plugin/common-services/DOLPHIN/1.3.6/configuration/dolphin-common.xml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (chengshiwen) [inline ambari_plugin/common-services/DOLPHIN/1.3.6/configuration/dolphin-common.xml:1]: One more empty line

**Review by chengshiwen — CHANGES_REQUESTED** [review]

**Review by chengshiwen — APPROVED** [review]
+1

**Comment by xingchun-chen** [comment]
+1

**Pull request merged**